packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts

```diff
@@ -1176,11 +1176,8 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
         await livenessStreamProvider!.dispatchStopVideoEvent();
       },
       async getLiveness(context) {
-        const { livenessStreamProvider } = context;
         const { onAnalysisComplete } = context.componentProps!;
 
-        livenessStreamProvider!.endStream();
```

Comment:
Just for my own understanding, was this in place in to handle a scenario where the stream was still active from a previous attempt?

Reply:
This was just in place so that we could ensure that the websocket connection was closed and that our input stream was closed.

But considering the websocket connection gets closed by the server we don't need to call this as at this point we have already stopped recording video.